### ISSUE TITLE
the permissions link talks about firefox and browser.

### ISSUE THREAD

**Issue by MadMattAu** [thundernest/addons-server#118]
The permissio0ns link makes reference to firewfox not Thunderbird and discusses the browser context.  This might be appropriate,  but confusing to users who have no concept of a browser contect in Thunderbird,  on in web browsers.

![image](https://user-images.githubusercontent.com/5334203/69203455-46da5300-0b94-11ea-865b-9b07d70684aa.png)

**Comment by Sancus** [comment]
Yes, unfortunately changing strings breaks localizations and that's much worse than having Firefox references, because we can't relocalize this site, and it's not under active development nor are there any plans to add localization capability any time soon.
